integration-test/rollingupdate-kubernetes/src/main/scala/akka/cluster/bootstrap/PodDeletionCostDemoApp.scala

```diff
@@ -23,6 +23,8 @@ object PodDeletionCostDemoApp extends App {
 
   AkkaManagement(system).start()
   ClusterBootstrap(system).start()
+
+  AppVersionRevision(system).start()
```

Comment:
shouldn't be a problem in practise, but theoretically the AppVersionRevision should be started before ClusterBootstrap. Call to `Cluster(system).setAppVersionLater` should be before joining.

Reply:
Fine to me, if that works with the config-based loading of this extension.